src/NuGet.Services.Staging.Manager/Filters/OwnerFilter.cs

```diff
@@ -1,37 +1,37 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
-using Microsoft.AspNet.Http;
-using Microsoft.AspNet.Mvc;
-using Microsoft.AspNet.Mvc.Filters;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
 
 namespace NuGet.Services.Staging.Manager.Filters
 {
-    public class OwnerFilter : ActionFilterAttribute
+    public class OwnerFilterServiceAttribute : ServiceFilterAttribute

```

Comment:
fixed

---
In reply to: [62967234](https://github.com/NuGet/NuGet.Staging/pull/44#discussion_r62967234) [](ancestors = 62967234)